src/main/kotlin/no/nav/tilleggsstonader/sak/behandling/fakta/BehandlingFaktaDto.kt

```diff
@@ -18,6 +18,7 @@ data class BehandlingFaktaDto(
     val aktivitet: FaktaAktivtet,
     val barn: List<FaktaBarn>,
     val dokumentasjon: FaktaDokumentasjon?,
+    val arena: ArenaFakta?,
```

Comment:
Kunne nesten ligget med VilkårperiodeGrunnlag. Synes det blir litt samme greie i hvert fall 🤷  Og kan sikkert argumentere for at VilkårperiodeGrunnlag burde vært her.

Reply:
Hehe ja, burde kanskje kunne ha VilkårperiodeGrunnlag her sånn at vi kun har 1 grunnlag